packages/devtools_app/lib/src/vm_service_wrapper.dart

```diff
@@ -213,7 +213,7 @@ class VmServiceWrapper implements VmService {
     if (await isProtocolVersionSupported(
         supportedVersion: SemanticVersion(major: 3, minor: 18))) {
       return trackFuture(
-        'getAllocationProfile',
+        '_getAllocationProfile',
```

Comment:
Why is this being changed to `_getAllocationProfile`? The current string references the existing public RPC.

Reply:
It was accidentally changed - shouldn't have been.